tests/unit/test-cmd/test.lint.js

```diff
@@ -66,6 +67,14 @@ describe('lint', () => {
     });
   });
 
+  it('allow deactivate runAsBinary param into linter', () => {
```

Comment:
I suggest re-wording this to:

it... passes shouldExitProgram option to the linter